README.md

```diff
@@ -109,6 +109,10 @@ The **#matrix** project has some environment variables that important to define.
 		   }
 		 ]
 
+Other option is to have a remote rooms config file. You can to configure a `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:
```

Comment:
```suggestion
Another option is to have a remote rooms config file (this file needs to be accessible via http/s). You can configure a `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:
```

Reply:
@Verhaeg what do you think?

Reply:
Great! 
and awesome addition to the repository!